hooks/useAuctionActions.js

```diff
@@ -5,16 +5,27 @@ const useAuctionActions = () => {
   const { maker, web3Connected } = useMaker();
 
   //ETH tend
-  async function callTend(auctionId, lotSize, bidAmount) {
+  function callEthTend(auctionId, lotSize, bidAmount) {
```

Comment:
Probably a single method that might accept `({auctionId, type}, lotSize, bidAmount)` or something like this.
Base on the `type` you can call the `flipEth` or `flipBat`.

What do you think about the following:

```javascript
function callTend(auctionid, colType, lotSize, bidAmount) {
  switch(colType) {
   case COLLATERAL_TYPES.ETH_A: return maker .service(AUCTION_DATA_FETCHER)....;
   case COLLATERAL_TYPES.BAT_A":return maker.service(AICTION_DATA_FETCHER)...;
   default throw Error("Unknown Collateral");
  }
}`

Reply:
In a perfect world we will need a `factory method` that will return a `tendCall` based on the collateral type.

Reply:
yes good call, tried to simplify them a bit here.